spec/spec_helper.rb

```diff
@@ -1,2 +1 @@
-require 'rubygems'
```

Comment:
http://www.rubyinside.com/why-using-require-rubygems-is-wrong-1478.html

- I can accept this change..